installer/arch/prepare

```diff
@@ -0,0 +1,244 @@
+            params="--asdeps"
+        else
+            break
+        fi
+        shift
+    done
+    while [ "$#" != 0 ]; do
+        if [ "$1" = '--' ]; then
+            shift
+            break
+        fi
+        aurstrip=${1#aur:}
+        if [ "$aurstrip" = "$1" ]; then
+            if [ -n "$multilib" ] && \
+	       [ "$1" = "gcc" ]; then
+                pkgs="$pkgs $1-multilib"
+            else
+                pkgs="$pkgs $1"
+            fi
+        else
+            aurpkgs="$aurpkgs $aurstrip"
+        fi
+        shift
+    done
+    if [ -n "$pkgs" ]; then
+        pacman -S --noconfirm --needed $params $pkgs
+    fi
+
+    if [ -n "$aurpkgs" ]; then
+        for aurpkg in $aurpkgs; do
+            aurcompile $aurpkg $params
+        done
+    fi
+}
+
+
+# install_pkg_dist: see install_pkg() in prepare.sh for details.
+install_pkg_dist() {
+    error 1 'ERROR: install_pkg_dist is not supported on Arch'
+}
+
+
+# remove_dist: see remove() in prepare.sh for details.
+remove_dist() {
+    if [ "$#" -gt 0 ]; then
+        pacman -R "$@"
+    fi
+}
+
+
+# list_uninstalled_dist: see list_uninstalled() in prepare.sh for details.
+list_uninstalled_dist() {
+    local suffix="$1" pkg
+    if [ -n "$suffix" ]; then
+        error 1 'list_uninstalled_dist: suffix makes no sense on Arch'
+    fi
+    shift
+    for pkg in "$@"; do
+        if ! pacman -Q "$pkg" 2>/dev/null >/dev/null; then
+            echo -n "$pkg "
+        fi
+    done
+}
+
+# Make sure package architecture is set manually.
+# Using the default Auto here detects the chroot from uname -m, breaking i686
+# chroots on x86-64.
+sed -i 's/^[ \t]*Architecture[ \t].*/Architecture = '"$ARCH"'/' /etc/pacman.conf
+
+# Run boostrap second stage if it hasn't already happened
+bootstrappkglist="/crouton-bootstrap-pkg"
+if [ -r "$bootstrappkglist" ]; then
+    # This is technically not necessary, as we are not fetching anything at this
+    # stage, but pacman complains if no mirror is set. We setup a dummy one,
+    # to make sure we really do not fetch anything (we need to install the
+    # exact same packages as during the bootstrap creation, to make sure we
+    # do not leave any cruft behind).
+    if [ -n "$MIRROR" -a "$MIRROR" != 'unspecified' ]; then
+        echo "Server = http://www.example.com/" > "/etc/pacman.d/mirrorlist"
+    fi
+
+    # Force overwritting packages files in the bootstrap, without any sort
+    # of dependency checking: this makes sure package content are properly
+    # recorded
+    pacman -S --nodeps --nodeps --asdeps --noconfirm --force \
+        `cat "$bootstrappkglist"`
+
+    # Reset architecture
+    sed -i 's/^[ \t]*Architecture[ \t].*/Architecture = '"$ARCH"'/' \
+        /etc/pacman.conf
+
+    # Restore the original /etc/resolv.conf
+    if [ -f /etc/resolv.conf.pacorig ]; then
+        rm -f /etc/resolv.conf
+        mv -f /etc/resolv.conf.pacorig /etc/resolv.conf
+    fi
+
+    # Initialize pacman keyring (i686/x86_64 only)
```

Comment:
Actually, there is. It's just that it's not enabled by default (http://archlinuxarm.org/about/package-signing).

Added support.